layers/+frameworks/ruby-on-rails/packages.el

```diff
@@ -64,7 +64,6 @@
         "fgz" 'projectile-rails-goto-spec-helper
         "fg." 'projectile-rails-goto-file-at-point
         ;; Rails external commands
-        "f:" 'projectile-rails-rake
```

Comment:
I have compared the commands `projectile-rails-rake` and `rake` and noticed that the first is outputting to a `projectile-rake` specific buffer and the latter is not.

I am no ruby developer so I do not know if this makes any difference, could you quickly test this?
If they are the same the keybinding can be removed however you also need to remove the documentation for the keybinding in `ruby-on-rails` layer.

Reply:
Thanks, I didn't notice. I'm making this wip not to be merged.

Reply:
@smile13241324 
Since this PR aims to just add missing prefixes, I decided not to remove this. Thank you.